core/src/main/java/com/linecorp/armeria/server/annotation/RequestObject.java

```diff
@@ -22,18 +22,51 @@
 import java.lang.annotation.Target;
 
 /**
- * Specifies which parameter should be converted by {@link RequestConverterFunction}.
+ * Specifies which element should be converted by {@link RequestConverterFunction}.
  *
+ * <p>A {@link RequestObject} can be implicitly applied when:
+ * <ul>
+ *     <li>A parameter is neither annotated nor automatically injected type in an annotated method.</li>
+ *     <li>A {@link RequestConverter} annotation is specified on a parameter in an annotated method
+ *     or a setter of a bean.</li>
+ *     <li>A {@link RequestConverter} annotation is specified on a field of a bean.</li>
+ *     <li>A {@link RequestConverter} annotation is specified on a method or constructor which has
+ *     only one parameter.</li>
+ * </ul>
+ * The following example shows when to apply a {@link RequestObject} implicitly.
```

Comment:
explicitly?

Reply:
Please refer to https://github.com/line/armeria/pull/1412#discussion_r230257679